apps/opik-frontend/src/components/pages-shared/automations/AddEditRuleDialog/schema.ts

```diff
@@ -16,15 +16,22 @@ import {
 } from "@/types/llm";
 import { generateRandomString } from "@/lib/utils";
 import { COLUMN_TYPE } from "@/types/shared";
-import { hasImagesInContent } from "@/lib/llm";
-import { supportsImageInput } from "@/lib/modelCapabilities";
+import { hasImagesInContent, hasVideosInContent } from "@/lib/llm";
+import {
+  supportsImageInput,
+  supportsVideoInput,
+} from "@/lib/modelCapabilities";
 import { PROVIDER_MODELS } from "@/hooks/useLLMProviderModelsData";
 
 const isOpenAIModel = (modelName: string): boolean => {
   const openAIModels = PROVIDER_MODELS[PROVIDER_TYPE.OPEN_AI] || [];
   return openAIModels.some((model) => model.value === modelName);
 };
 
+const isVllmQwenModel = (modelName: string): boolean => {
```

Comment:
isVllmQwenModel should NOT be required as supportsVideoInput should detect the qwn models with regex for `.*vl.*`